### ISSUE TITLE
about experiment result

### ISSUE THREAD

**Issue by kengkeng1** [qizekun/ReCon#6]
Hi，I download your pretain model from the google cloud and finetune on NVIDIA 3090，and achieve the 92.38% result on svm modelnet40 task ，and  94.49% 92.6% 89.62% on the ScanObjectNN task, and the random seed also be set to 0 .

Is it related to the server and Pytorch environment I'm using? Or I need to cancel the setting of the random seed and run multiple times.

Thank you！

**Comment by qizekun** [comment]
For classification downstream tasks, we randomly select 8 seeds to obtain the best checkpoint. The best checkpoints and logs have been uploaded to [Google Drive](https://drive.google.com/drive/folders/1s9sZmT6XC_uTdKP-xUrl0a_gsSNSiEVf?usp=drive_link). Besides, you can use the voting strategy in classification testing to reproduce the performance reported in the paper. 

For a quick test, you could run `bash scripts/test.sh <GPU> <exp_name> <path/to/best/fine-tuned/model>` The best checkpoints are from [Google Drive](https://drive.google.com/drive/folders/1s9sZmT6XC_uTdKP-xUrl0a_gsSNSiEVf?usp=drive_link).

**Comment by kengkeng1** [comment]
Thank you for explaining patiently. It means that when get a pretain model ，test 250 epoch、275 epoch、300 epoch models 8 times separately for w/o voting strategy result. Then take the best finetune model to get w/ voting strategy result in 8 times?

**Comment by qizekun** [comment]
Yes, that is. Because this is the only way to reproduce the [baseline](https://github.com/Pang-Yatian/Point-MAE), we also adopt this approach.

**Comment by kengkeng1** [comment]
Thank you， I get it.  And if the pretain-model also train several times or not ?

**Comment by qizekun** [comment]
No, only once.

**Comment by kengkeng1** [comment]
Thank you for your response!

**Comment by Amazingren** [comment]
原来如此啊
